libraries/SITL/SITL.cpp

```diff
@@ -47,6 +47,15 @@ extern const AP_HAL::HAL& hal;
 #endif
 #endif
 
+#if (CONFIG_HAL_BOARD != HAL_BOARD_SITL)
+// For on-hardware, set allowed relay channels to zero.
+// Requires user to change the param to allow hadware access.
+#define SIM_DEFAULT_ENABLED_RELAY_CHANNELS 0
+#else
+// For SITL, set allowed relay channels to the full mask.
+#define SIM_DEFAULT_ENABLED_RELAY_CHANNELS (1<<16)-1
```

Comment:
You have other options here:
```suggestion
#define SIM_DEFAULT_ENABLED_RELAY_CHANNELS UINT16_MAX
```
or
```suggestion
#define SIM_DEFAULT_ENABLED_RELAY_CHANNELS 0xFFFF
```
or
```suggestion
#define SIM_DEFAULT_ENABLED_RELAY_CHANNELS 0b1111111111111111
```

... all of which I think look a little better than the maths :-)

This'll do, 'though

Reply:
Updated to remove maths.